packages/nc-gui-v2/composables/useViewColumns.ts

```diff
@@ -124,6 +125,12 @@ export default function (
       []) as ColumnType[]
   })
 
+  // reload view columns when table meta changes
+  watch(
+    () => meta?.value,
+    () => loadViewColumns(),
+  )
```

Comment:
pretty sure this will never get triggered

Reply:
It will trigger whenever some meta changes happen ( for eg, new column addition )  we are reloading meta even if cache exists.